frontend/src/App.js

```diff
@@ -89,6 +92,16 @@ export default class App extends React.Component {
                 component: () => <h1>{pageTitles.editEntries}</h1>,
                 private: true,
             },
+            {
+                path: '/users/:userId',
```

Comment:
Y no substitue? :userId

Reply:
I think doing that after we write selectors for getting user data would be better.